src/Esfa.Vacancy.Register.Infrastructure/Services/ApprenticeshipSearchService.cs

```diff
@@ -43,6 +43,8 @@ public ApprenticeshipSearchService(IProvideSettings provideSettings, ILog logger
 
             ISearchResponse<ApprenticeshipSummary> esReponse;
 
+            _logger.Info($"Querying Elastic Index with following parameters: {parameters}");
```

Comment:
You may want to put the index name in this log message. Just for convenience of differentiating if the Traineeship search is on a different index and being able to see clearly from the message. A suggestion because maybe its isn't needed.